main-project/src/members/entity/host-members.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Meeting } from 'src/meetings/entity/meeting.entity';
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('host_members')
+export class HostMembers extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @Column()
+  user_no: number;
+
+  @OneToOne((type) => Meeting, (meeting) => meeting.meetingHostMembers, {
+    nullable: false,
+  })
+  @JoinColumn({ name: 'meeting_no' })
+  meeting: Meeting;
```

Comment:
```suggestion
  @JoinColumn({ name: 'meeting_no' })
  meetingNo: Meeting;
```

외래키의 경우 이름을 맞춰주시는 게 추후 코드 작성하실 때 혼동을 줄일 수 있습니다!

Reply:
변수명 컨벤션에 맞춰서 수정하고 EntityRepository 파일 별로 분리했어요! 피드백 감사합니다 :)